updated_main.py

```diff
@@ -3,57 +3,86 @@
 import bs4
```

Comment:
* Lets give this file a better name.
* Add a docstring on top of this file to document how to use/invoke this script.